editor/src/clj/editor/defold_project.clj

```diff
@@ -405,11 +395,14 @@
                       render-nested-progress! (progress/nest-render-progress render-progress! parent-progress task-size)]
                   (pair (+ accumulated-task-size task-size)
                         (conj progress-fns
-                              (fn progress-fn [loaded-node-index proj-path]
-                                (render-nested-progress!
-                                  (progress/make (str task-label \space proj-path)
-                                                 loaded-node-count
-                                                 loaded-node-index)))))))
+                              (fn progress-fn
+                                ([progress]
+                                 (render-progress! progress))
+                                ([loaded-node-index proj-path]
+                                 (render-nested-progress!
+                                   (progress/make (str task-label \space proj-path)
+                                                  loaded-node-count
+                                                  loaded-node-index))))))))
```

Comment:
Changed the `progress-fn` so it can also be called with a single `progress` object. We use it to go into the indeterminate progress state at the end of processing all the `tx-data`, but not yet updating successors and so on.